src/lmbin/arpa2fst.cc

```diff
@@ -0,0 +1,129 @@
+// bin/arpa2fst.cc
+//
+// Copyright 2009-2011  Gilles Boulianne.
+//
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "lm/arpa-lm-compiler.h"
+#include "util/kaldi-io.h"
+#include "util/parse-options.h"
+
+
+int main(int argc, char *argv[]) {
+  using namespace kaldi;
+  try {
+    const char *usage  =
+        "Converts an ARPA format language model into a FST\n"
+        "Usage: arpa2fst [opts] (input_arpa|-)  [output_fst|-]\n";
```

Comment:
inconsistent use of square versus round brackets.  and <input-arpa> and <output-fst> should be written in angle brackets.  (I know the code was old).